boofuzz/fuzz_logger_csv.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function
+import sys
+import time
+import datetime
+import csv
+
+from . import helpers
+from . import ifuzz_logger_backend
+
+# from IPython import embed
+
+# init()
+
+def hex_to_hexstr(input_bytes):
+    """
+    Render input_bytes as ASCII-encoded hex bytes, followed by a best effort
+    utf-8 rendering.
+
+    @param input_bytes: Arbitrary bytes.
+
+    @return: Printable string.
+    """
+    #return helpers.hex_str(input_bytes) + " " + repr(bytes(input_bytes))
+    return '0x' + ''.join("{:02x}".format(ord(b)) for b in input_bytes)
+
+DEFAULT_HEX_TO_STR = hex_to_hexstr
+
+
+def get_time_stamp():
+    # t = time.time()
+    # s = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(t))
+    # s += ",%03d]" % (t * 1000 % 1000)
+    # s = t.isoFormat()
+    s = datetime.datetime.utcnow().isoformat()
```

Comment:
Nice! An ISO format sounds better than the made up one we had...

Remove commented out timestamp method.